app/src/pages/ActivitiesPages/MyOfferedHelp/MyOfferHelpDescription/index.jsx

```diff
@@ -168,22 +154,27 @@ export default function OfferHelpDescription({ route, navigation }) {
         else return renderWaitingHelpOwnerMessage();
     };
 
+    const getAllPossibleHelpedUsers = () => {
+        const fakePossibleHelpedUsers = help.possibleHelpedUsers;
+        const fakePossibleEntities = help.possibleEntities;
+        fakePossibleEntities.push(...fakePossibleHelpedUsers);
+
+        return fakePossibleEntities;
+    };
+
```

Comment:
Só uma sugestão pra deixar o código mais legível, não é obrigatória.

```suggestion
    const getAllPossibleHelpedUsers = () => {
       const { possibleHelpedUsers, possibleEntities } = help;
       return [...possibleHelpedUsers, ...possibleEntities];
    };

```